src/InsertionsClient/ConsoleApp/Program.cs

```diff
@@ -152,10 +152,13 @@ private static void ShowResults(UpdateResults results)
 					}
 				}
 
-				Trace.WriteLine($"{results.PropsFileUpdateResults.UnrecognizedVariables.Count} variables were not found in props files:");
-				foreach (var variable in results.PropsFileUpdateResults.UnrecognizedVariables)
+				if(results.PropsFileUpdateResults.UnrecognizedVariables.Count != 0)
 				{
-					Trace.WriteLine($"        {variable.Name} in {variable.ReferencedFilePath}");
+					Trace.WriteLine($"{results.PropsFileUpdateResults.UnrecognizedVariables.Count} variables were not found in props files:");
+					foreach (var variable in results.PropsFileUpdateResults.UnrecognizedVariables)
+					{
+						Trace.WriteLine($"        {variable.Name} in {variable.ReferencedFilePath}");

```

Comment:
minor: not a big fan of the empty space. Is it mainly for the console's output's benefit?

Reply:
Yes, purely for the visual benefit. This was the approach we took in the first version of the code, I just followed along.